### DISCUSSION TITLE
Double SplashScreen on Android 12

### DISCUSSION THREAD

**Discussion by minghwee** [timbru31/cordova-plugin-lottie-splashscreen#458] [Q&A]
Hi

Would like to check for Android 12, there's currently a default splash screen generated by android. Would there by any workaround to disable it? Or could we control the background color of the default splash screen generated by android?

Best Regards

**Comment by timbru31** [comment]
The SplashScreen API of Android 12 is all new to me, too. I need to actually deep dive into this and then come up with a solution.
I'll let this question open until I gathered more evidence/fixes/workarounds/.... :)

**Reply by minghwee** [reply]
thanks @timbru31

**Reply by On1ty** [reply]
Good day! Any update on this?
